modules/hosted-control-planes-monitoring-dashboard.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="hosted-control-planes-monitoring-dashboard_{context}"]
+= Creating monitoring dashboards for hosted clusters
+
+The HyperShift Operator can create or delete monitoring dashboards in the management cluster for each hosted cluster that it manages.
+
+//lahinson - oct. 2023 - The following paragraph was in Cesar's original draft. Does it still apply?
```

Comment:
@rokej Can you take a look at the text on line 13 and let me know if it still applies now that several changes have been made to this file?

Reply:
Ah. Forgot to comment that we should remove line 13.

Reply:
@rokej Will do. Thanks for confirming.

Reply:
Sorry, just delete line 13 and add the paragraph I mentioned in line 39